File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.java
class Solution {
    public int countSeniors(String[] details) 
    {
        int count=0;
        for(int i=0;i<details.length;i++)
        {
            int a=Integer.parseInt(details[i].substring(11,13));
            if(a>60)
            count++;
        }
        return count;
    }
}

File: 2727-number-of-senior-citizens/2727-number-of-senior-citizens.js
/**
 * @param {string[]} details
 * @return {number}
 */
var countSeniors = function(details) 
{
        let count=0;
        for(let i=0;i<details.length;i++)
        {
            // console.log(details[i].substring(11,13))
            // let a=parseInt(details[i].substring(11,13))
            if(details[i].substring(11,13)>60)
            {count++;}

        }
        return count;
};